### ISSUE TITLE
Malfunctioning cmdlet for New-TenantAllowBlockListItems

### ISSUE THREAD

**Issue by avatar7008** [MicrosoftDocs/office-docs-powershell#9923]
The -Allow parameter does not work correctly, returns with the error:

A parameter cannot be found that matches parameter name "Allow".
    + CategoryInfo              :  InvalidArgument: (:) [New-TenantAllowBlockListItems], ParameterBindingException
    + FullyQualifiedErrorId  :  NamedParameterNotFound,New-TenantAllowBlockListItems
    + PSComputerName     :  aus01b.ps.compliance.protection.outlook.com

According to the KB → https://learn.microsoft.com/powershell/module/exchange/new-tenantallowblocklistitems
You don't need to specify a value with the -Allow switch but it results in the error above. 

Steps to reproduce:

1. Connect-ExchangeOnline
2. Connect-IPPSsession -UserPrincipalName "globaladminuser"
3. New-TenantAllowBlockListItems -Allow -ListType Url -ListSubType AdvancedDelivery -Entries "url1","url2" -NoExpiration

**Comment by chrisda** [comment]
@avatar7008, although the cmdlet is available in Exchange Online PowerShell and Security & Compliance PowerShell, the Allow parameter is available only in Exchange Online PowerShell. Although the Applicable: value for the parameter stated this, but I realize that's subtle, so I added the following text to the beginning of the description:

> This parameter is available only in Exchange Online PowerShell.

I also did this for the ListSubType and SubmissionID parameters, which have the same issue.

OK?
